lumibot/data_sources/pandas_data.py

```diff
@@ -289,9 +289,11 @@ def _pull_source_symbol_bars(
             )
         # Return None if data.get_bars returns a ValueError
         except ValueError as e:
-            raise ValueError(f"Error getting bars for {asset}: {e}")
+            print(f"Error getting bars for {asset}: {e}")
+            return None
```

Comment:
I think that this print should be a logger.info() message instead so that it makes it into log files during runs.  Might get missed in the console prints.